services/order.py

```diff
@@ -0,0 +1,31 @@
+from django.db import transaction
+from django.contrib.auth import get_user_model
+
+from db.models import Ticket, Order
+
+
+def create_order(tickets, username, date=None):
+    with transaction.atomic():
+
+        order = Order.objects.create(
+            user=get_user_model().objects.get(username=username)
+        )
+
+        if date:
+            order.created_at = date
+
+        order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(movie_session_id=ticket["movie_session"],
+                                  row=ticket["row"],
+                                  seat=ticket["seat"],
+                                  order=order,
+                                  )
```

Comment:
Bad code style. Either move the bracket to the preceding line or write all arguments on a new line.